changelogs/fragments/blowfish_ident.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
```

I think this is a new feature rather than a bugfix.
```suggestion
minor_changes:
```